app/routes/login.tsx

```diff
@@ -15,10 +15,45 @@ export const action = async ({ request }: ActionArgs) => {
 
 export default function Login() {
   return (
-    <Form method="post">
-      <Heading>Login</Heading>
-      <Text>You will be logged in (a session will be created for you)</Text>
-      <Button type="submit">Login</Button>
-    </Form>
+    <Center height="container.md" marginTop="28">
```

Comment:
`h=... mt=...`

Reply:
I think using the actual word as to the key characters is easier for readability

Reply:
I kind of like the waterfall aesthetic, makes it looks proportional to the container, not sure though we can ask the others for input